.github/continuous-integration.yml

```diff
@@ -0,0 +1,40 @@
+# https://help.github.com/en/categories/automating-your-workflow-with-github-actions
+
+name: "Continuous Integration"
```

Comment:
If you prefer a different name for the workflow, let me know.

The name will be reflected on the badge.